src/ModuleList/Quiz/styled.ts

```diff
@@ -1,33 +1,37 @@
+import { Button, Text } from '@chakra-ui/core';
 import styled from '@emotion/styled';
 
-export const SubmitQuiz = styled('a')`
+export const QuizScore = styled(Text)`
+  color: ${({ percent, minScore }:{percent: number, minScore: number}) => (
+    percent >= minScore ? '#39a139' : '#ff5151'
+  )};
+  margin: 1.5rem auto;
+`;
+
+export const SubmitQuiz = styled(Button)`
   margin: 0 auto;
-  padding: 5px 17px;
-  border-radius: 3.5px;
-  background-color: #4195A5;
-  color: #FFFFFF;
-  cursor: pointer;
-  &:hover {
-    opacity: 0.2;
-  }
 `;
 
 export const CheckIcon = styled('p')`
+  margin-right: 1rem;
   &:after {
     content: '✔';
     color: #90EE90;
   }
 `;
 
 export const IncorrectIcon = styled('p')`
+  margin-right: 1rem;
   &:after {
     content: '✘';
     color: red;
   }
 `;
 
-export const QuestionLabel = styled.label`
-  color: ${(props:{isIncorrect: boolean}) => props.isIncorrect ? '#C5A3B4' : '#FFFFFF'};
+export const QuestionLabel = styled(Text)`
+  color: ${({ isIncorrect }:{isIncorrect: boolean}) => (
+    isIncorrect ? '#C5A3B4' : '#FFFFFF'
+  )};
```

Comment:
Shouldn't we hide the correct answers to people who want to retry?
Maybe we should hide them until the users click the "I'm Done" button.

Reply:
Hmm... Maybe? I feel like that requires more discussion. Personally, I would want to review the questions that I got wrong so that I can review them before I retry but maybe that doesn't aid in learning. @takakonishimura What are your thoughts?